app/Http/Controllers/AuthenticationController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Mail;
+use Illuminate\Support\Str;
+use App\Enums\AuthenticationStatus;
+use App\Models\Authentication;
+use App\Mail\SendTokenMail;
+use App\Http\Requests\AuthenticationRequest;
+use Carbon\Carbon;
+
+class AuthenticationController extends Controller
+{
+    public function create(Request $request)
+    {
+        return view('authentication.create');
+    }
+
+    public function store(AuthenticationRequest $request)
+    {
+        $user_token = Str::random(rand(30, 50));
+        $authentication = Authentication::where('email', $request->email)
+            ->where('status', AuthenticationStatus::MAIL_SENT)
+            ->first();
+
+        $expired_at = Carbon::now()->addMinutes(15);
```

Comment:
```suggestion
        $expired_at = now()->addMinutes(15);
```
今回は修正不要ですが、現在の日時を取得する際はnow()を使います。
こちらはLaravelはヘルパー関数があり、現在の日時でCarbonクラスのインスタンスを返却します。

Reply:
ご指摘ありがとうございます。
ヘルパ関数を使用していくようにして参ります。